components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -3692,5 +3249,628 @@ subroutine get_time_space_phys_variables( &
 
 end subroutine get_time_space_phys_variables
 
+subroutine cloud_sedimentation(kts,kte,    &
+   qc_incld,rho,inv_rho,lcldm,acn,inv_dzq,&
+   dt,odt,dnu,lammin,lammax,log_predictNc, & 
+   qc, nc, nc_incld,mu_c,nu,lamc,prt_liq,qc_tend,nc_tend) 
+   
+   implicit none 
+   integer, intent(in) :: kts, kte
+
+   real(rtype), intent(in), dimension(kts:kte) :: qc_incld
+
+   real(rtype), intent(in), dimension(kts:kte) :: rho
+   real(rtype), intent(in), dimension(kts:kte) :: inv_rho 
+   real(rtype), intent(in), dimension(kts:kte) :: lcldm 
+   real(rtype), intent(in), dimension(kts:kte) :: acn 
+   real(rtype), intent(in), dimension(kts:kte) :: inv_dzq
+   real(rtype), intent(in) :: dt 
+   real(rtype), intent(in) :: odt 
+   real(rtype), dimension(:), intent(in) :: dnu
+   real(rtype), intent(in) :: lammin
+   real(rtype), intent(in) :: lammax 
+   logical, intent(in) :: log_predictNc 
+
+   real(rtype), intent(inout), dimension(kts:kte) :: qc 
+   real(rtype), intent(inout), dimension(kts:kte) :: nc 
+   real(rtype), intent(inout), dimension(kts:kte) :: nc_incld 
+   real(rtype), intent(inout), dimension(kts:kte) :: mu_c 
+   real(rtype), intent(inout), dimension(kts:kte) :: nu
+   real(rtype), intent(inout), dimension(kts:kte) :: lamc
+   real(rtype), intent(inout) :: prt_liq
+   real(rtype), intent(inout), dimension(kts:kte) :: qc_tend 
+   real(rtype), intent(inout), dimension(kts:kte) :: nc_tend 
+   
+   logical :: log_qxpresent, qc_present, two_moment
```

Comment:
Intel compiler (maybe others) doesn't like when the loop name has been declared as a logical ahead of time.  We should remove `qc_present` and `two_moment` here.